java/ql/src/Security/CWE/CWE-200/TempDirLocalInformationDisclosureFromSystemProperty.ql

```diff
@@ -0,0 +1,94 @@
+/**
+ * @name Temporary Directory Local information disclosure
+ * @description Detect local information disclosure via the java temporary directory
```

Comment:
This description still needs an update. As Felicity wrote, the grammatical structure should be along the lines of "Doing _this-thing-the-query-detects_ causes/may cause _something-bad_", i.e. the description from the other query is likely also fine here.
```suggestion
 * @description Writing information without explicit permissions to a shared temporary directory may disclose it to other users.
```